Sources/SecureXPC/Server/XPCServer.swift

```diff
@@ -63,152 +67,91 @@ import Foundation
 /// ## Topics
 /// ### Creating a Server
 /// - ``forBlessedHelperTool()``
-/// - ``init(machServiceName:clientRequirements:)``
+/// - ``forMachService(named:clientRequirements:)``
+/// - ``forThisXPCService()``
 ///
 /// ### Registering Routes
-/// - ``registerRoute(_:handler:)-8ydqa``
-/// - ``registerRoute(_:handler:)-76t5b``
-/// - ``registerRoute(_:handler:)-39dgn``
-/// - ``registerRoute(_:handler:)-1rnkw``
+/// - ``registerRoute(_:handler:)-1jw9d``
+/// - ``registerRoute(_:handler:)-4fxv0``
+/// - ``registerRoute(_:handler:)-4ttqe``
+/// - ``registerRoute(_:handler:)-9a0x9``
 ///
 /// ### Starting a Server
 /// - ``start()``
 ///
 /// ### Error Handling
 /// - ``errorHandler``
-public class XPCMachServer {
-    
+public class XPCServer {
+
+	// MARK: Public factories
+
+	/// Creates a server that accepts requests from clients which meet the security requirements.
+	///
+	/// Because many processes on the system can talk to an XPC Mach Service, when creating a server it is required that you specifiy the security requirements
+	/// of any connecting clients:
+	/// ```swift
+	/// let reqString = """identifier "com.example.AuthorizedClient" and certificate leaf[subject.OU] = "4L0ZG128MM" """
+	/// var requirement: SecRequirement?
+	/// if SecRequirementCreateWithString(reqString as CFString,
+	///                                   SecCSFlags(),
+	///                                   &requirement) == errSecSuccess,
+	///   let requirement = requirement {
+	///    let server = XPCMachServer(machServiceName: "com.example.service",
```

Comment:
This is now out of date. I can address this when updating documentation after this has been merged.